### PR TITLE
Create ochestrator cookbook

### PR DESCRIPTION
**Pull request by Seitanas**
@vinted/sre

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name 'chef-orchestrator'
```
Comment (tomas-didziokas) [inline metadata.rb]: maybe just orchestrator ?

Comment (Seitanas) [reply metadata.rb]: @tomas-didziokas  - good point, fixed.

Comment (ernestas-vinted) [reply metadata.rb]: indeed :+1: `orchestartor` is available https://supermarket.chef.io/cookbooks?utf8=%E2%9C%93&q=orchestrator&platforms%5B%5D=

File: libraries/orchestrator.rb [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+          end
+        end
+      end
+
+      def orchestrator_system_user
+        group node['orchestrator']['group']
+
+        user node['orchestrator']['user'] do
+          comment 'Orchestrator daemon user'
+          group node['orchestrator']['group']
```
Comment (ernestas-vinted) [inline libraries/orchestrator.rb:240]: shell '/bin/false' ?

File: recipes/orchestrator.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+orchestrator_service 'default' do
```
Comment (ernestas-vinted) [inline recipes/orchestrator.rb:1]: remove empty recipes/default.rb

**Review by ernestas-vinted — CHANGES_REQUESTED** [review]

File: libraries/orchestrator.rb [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+        sqlite_db_install if new_resource.database_backend == 'sqlite'
+        orchestrator_config
+        orchestrator_service
+      end
+      # rubocop:enable Metrics/AbcSize
+
+      def install_mysql?
+        return true if new_resource.install_mysql && new_resource.database_backend == 'mysql'
+
+        false
```
Comment (ernestas-vinted) [inline libraries/orchestrator.rb]: you can remove false, function will return nil == false, by default

```
$ irb
irb(main):001:0> def true_or?
irb(main):002:1> return true if false
irb(main):003:1> end
=> :true_or?
irb(main):004:0> true_or?
=> nil
```

**Review by ernestas-vinted — CHANGES_REQUESTED** [review]

**Review by dmitry-g — APPROVED** [review]

File: libraries/orchestrator.rb (hunk 4 above)
Comment (Seitanas) [reply libraries/orchestrator.rb]: @ernestas-vinted  - fixed

**Review by ernestas-vinted — APPROVED** [review]

**Pull request merged**